static/refcache.json

```diff
@@ -1499,6 +1503,14 @@
     "StatusCode": 200,
     "LastSeen": "2023-02-20T08:02:57.433517-05:00"
   },
+  "https://github.com/census-instrumentation/opencensus-python": {
+    "StatusCode": 200,
+    "LastSeen": "2023-04-24T16:54:55.217650606Z"
+  },
+  "https://github.com/census-instrumentation/opencensus-python,": {
```

Comment:
Any idea why there are two entries for the same link?

Reply:
there is a comma in one of them and in the other not, maybe you remove that one line and redo the check

Reply:
OK I removed it and re-ran `npm test`. Seems to have fixed things, thank you!